### DISCUSSION TITLE
离线下载设置上传限速

### DISCUSSION THREAD

**Discussion by Jerry12228** [AlistGo/alist#3348] [Q&A]
离线下载文件功能中，下载完成后从本地上传到对应存储时，可以限制上传速度吗？

**Comment by f63531119** [comment]
aria2自带全局限速你是猴子派来的

**Reply by Jerry12228** [reply]
> aria2自带全局限速你是猴子派来的

请注意，问题中指的是aria2完成下载后，对由alist将下载完成的文件上传到存储（如阿里云盘）中时的上传速度进行限制，而不是限制aria2下载时分享的速度，请正确理解问题
